src/views/AgmExtension.vue

```diff
@@ -454,7 +454,7 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin,
         ret = await LegalServices.createFiling(this.getIdentifier, filing, isDraft)
       }
       return ret
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
By default, `error` is of type Unknown, which causes code errors on lines 459 and 460.